src/NServiceBus.RabbitMQ/Routing/ISupportNativeDelays.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace NServiceBus.Transport.RabbitMQ
+{
+    using global::RabbitMQ.Client;
+
+    /// <summary>
+    /// An extension for <see cref="IRoutingTopology"/> implementations which allows
+    /// for receiving messages from the delay infrastructure.
```

Comment:
@adamralph Yes, making that change will ripple through the code in a number of ways, including changing the name of the setting: `AllEndpointsSupportNativeDelays` should become `AllEndpointsSupportDelayedDelivery`.

That way we banish the term "native" everywhere.